src/main/java/com/arpnetworking/tsdcore/sinks/MetricsDataSerializer.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright 2020 Dropbox Inc.
```

Comment:
Probably can't reassign the copyright to this code given it's not contributed by Dropbox or modified significantly.

Reply:
One copy of this code lived in AggregationServerHttpSink.java, which does have a Dropbox copyright header.